### DISCUSSION TITLE
v8

### DISCUSSION THREAD

**Discussion by oae** [oae/gnome-shell-pano#73] [Announcements]
##### Documentation Changes

*  add gobject-introspection to build instructions (#70) (4c3ae531)

##### New Features

* **locales:**  update French translation (#67) (442d084d)
*  add primary selection sync. closes #30 (b57521f1)

##### Bug Fixes

*  change shortcut to global-shortcut, rename offline mode. closes #71 (44723379)

<hr /><em>This discussion was created from the release <a href='https://github.com/oae/gnome-shell-pano/releases/tag/v8'>v8</a>.</em>

**Comment by amadeusp** [comment]
Thank you so much for Pano, an amazing extension! I love it! 🤩
